tests/integration/components/polaris-dropzone-test.js

```diff
@@ -0,0 +1,576 @@
+import { render, triggerEvent, settled } from '@ember/test-helpers';
+import hbs from 'htmlbars-inline-precompile';
+import { capitalize, htmlSafe } from '@ember/string';
+import { selectFiles } from 'ember-native-dom-helpers';
+import buildNestedSelector from '../../helpers/build-nested-selector';
+import MockSvgJarComponent from '../../mocks/components/svg-jar';
+import MockEvent from '@smile-io/ember-polaris/test-support/mock-dropzone-event';
+
+module('Integration | Component | polaris-dropzone', function(hooks) {
+  setupRenderingTest(hooks);
+
+  const uploadedFiles = [{
+    name: 'jpeg file',
+    type: 'image/jpeg',
+  },
+  {
+    name: 'svg file',
+    type: 'image/svg',
+  },
+  {
+    name: 'pdf file',
+    type: 'application/pdf',
+  }];
+
+  const dropzoneSelector = '.Polaris-DropZone';
+  const containerSelector = buildNestedSelector(dropzoneSelector, '.Polaris-DropZone__Container');
+  // Hidden dropzone input element
+  const inputSelector = buildNestedSelector(
+    dropzoneSelector,
+    '.Polaris-VisuallyHidden',
+    'input'
+  );
+
+  // Overlays
+  const dropzoneOverlaySelector = buildNestedSelector(dropzoneSelector, '.Polaris-DropZone__Overlay');
+  const dropzoneOverlayStackSelector = buildNestedSelector(dropzoneOverlaySelector, '.Polaris-Stack');
+  const dropzoneOverlayStackItemSelector = buildNestedSelector(dropzoneOverlayStackSelector, '.Polaris-Stack__Item');
+  const dropzoneOverlayIconSelector = buildNestedSelector(dropzoneOverlayStackItemSelector, '.Polaris-Icon');
+  const dropzoneOverlayIconSVGSelector = buildNestedSelector(dropzoneOverlayIconSelector, 'svg');
+  const dropzoneOverlayTextSelector = buildNestedSelector(dropzoneOverlayStackItemSelector, '.Polaris-DisplayText');
+  const dropzoneOverlayCaptionSelector = buildNestedSelector(dropzoneOverlayStackItemSelector, '.Polaris-Caption');
+
+  test('it renders in non-block form', async function(assert) {
+    assert.expect(8);
+
+    await render(hbs`{{polaris-dropzone}}`);
+
+    // Defaults
+    assert.dom(dropzoneSelector).exists('has Polaris-DropZone class');
+    assert.dom(dropzoneSelector).hasClass('Polaris-DropZone--hasOutline', 'has outline class');
+    assert.dom(dropzoneSelector).hasClass('Polaris-DropZone--sizeLarge', 'has large size class');
+
+    assert.dom(containerSelector).exists('has the DropZone container element');
+
+    assert.dom(inputSelector).exists('has the input element hidden');
+    assert.dom(inputSelector).hasAttribute('autocomplete', 'off', 'input element has attribute autocomplete `off`');
+    assert.dom(inputSelector).hasAttribute('multiple', '', 'input element has attribute multiple');
+    assert.dom(inputSelector).hasAttribute('type', 'file', 'input element has attribute type `file`');
+  });
+
+  test('it supports `outline` property', async function(assert) {
+    assert.expect(2);
+
+    this.set('outline', true);
+
+    await render(hbs`{{polaris-dropzone outline=outline}}`);
```

Comment:
Add a check for this having the correct default value (i.e. when `outline` isn't set)?